Source/loadsave.cpp

```diff
@@ -2558,7 +2558,7 @@ void SaveLevel(SaveWriter &saveWriter)
 		}
 	}
 
-	file.WriteBE<int32_t>(ActiveMonsterCount);
+	file.WriteBE(static_cast<int32_t>(ActiveMonsterCount));

```

Comment:
would it work to instead change this to uint32_t?

Reply:
No, this is a narrowing warning (`ActiveMonsterCount` is a `size_t`).